### ISSUE TITLE
NTR: Toxoplasma gondii antibody titer high in CSF or blood (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8732]
New term request Toxoplasma gondii antibody titer high in CSF or blood (segal_101117150139): Cerebrospinal fluid

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000924
- label: Positive CSF Toxoplasma gondii antibody positivity
- parents: [Positive CSF pathogen-specific antibody positivity (HP:0430139)](https://hpo.jax.org/app/browse/term/HP:0430139)
- pmid: 
- def: The presence of antibodies in the cerebrospinal fluid (CSF) that react against a component of Toxoplasma gondii.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8732
- synonyms:
